agentcfg/fetch.go

```diff
@@ -91,9 +77,7 @@ func (f *Fetcher) request(r io.Reader, err error) ([]byte, error) {
 	}
 
 	result, err := ioutil.ReadAll(resp.Body)
-	if resp.StatusCode == http.StatusMultipleChoices {
```

Comment:
Did this change in Kibana so the check is not necessary anymore?

Reply:
i don't think it was ever needed?

Reply:
I think it was there from the beginning, so I expected Kibana could respond with it. If not needed, fine to remove.

Reply:
i don't recall either. the only thing i can think of is that the code is 300, so a check like `code < 300` tells you if a response is successful.